Orange/data/dask.py

```diff
@@ -150,6 +178,12 @@ def compute(self) -> Table:
         table.name = self.name
         return table
 
+    def sample(self, size: int = DEFAULT_SAMPLE_SIZE):
```

Comment:
Instead of size use something more concrete, like rows or instances. Perhaps the same function could also sample columns in the future. Or have a parameter that specifies the max bytesize.